test/modules/linear_boltzmann_solvers/transport_adjoint/adjoint_2d_1b_adjoint.lua

```diff
@@ -106,42 +89,31 @@ lbs_block =
     }
 }
 
-lbs_options =
-{
-    scattering_order = 1,
-}
+lbs_options = { scattering_order = 1 }
 
---############################################### Initialize and Execute Solver
-phys1 = lbs.DiscreteOrdinatesAdjointSolver.Create(lbs_block)
-lbs.SetOptions(phys1, lbs_options)
+phys = lbs.DiscreteOrdinatesAdjointSolver.Create(lbs_block)
+lbs.SetOptions(phys, lbs_options)
 
---############################################### Create QOIs
-tvol0 = NewRPP({xmin=2.3333,xmax=2.6666,ymin=4.16666,ymax=4.33333,infz=true})
-tvol1 = NewRPP({xmin=0.5   ,xmax=0.8333,ymin=4.16666,ymax=4.33333,infz=true})
+--############################################### Create QoI region
+qoi_vol = chi_mesh.RPPLogicalVolume.Create({ xmin = 0.5, xmax = 0.8333,
+                                             ymin = 4.16666, ymax = 4.33333,
+                                             infz = true })
 
-chiAdjointSolverAddResponseFunction(phys1,"QOI0",tvol0)
-chiAdjointSolverAddResponseFunction(phys1,"QOI1",tvol1)
-chiSolverSetBasicOption(phys1, "REFERENCE_RF", "QOI1")
+chiAdjointSolverAddResponseFunction(phys, "QoI", qoi_vol)
+chiSolverSetBasicOption(phys, "REFERENCE_RF", "QoI")
```

Comment:
what is REFERENCE_RF ?

Reply:
The syntax that has been used by the adjoint solver since Jan put it in the code. This will be deprecated in a future PR introducing an adjoint option into the main LBS solver.